.eslint-dictionary.json

```diff
@@ -50,6 +51,7 @@
   "birthdate",
   "bitwise",
   "bool",
+  "breakcirculardependency",
```

Comment:
If this is going to be added to the dictionary, please remove the eslint-disable flags in `packages/amplify-category-auth/src/provider-utils/awscloudformation/auth-stack-builder/auth-stack-transform.ts` and `packages/amplify-category-auth/src/provider-utils/awscloudformation/auth-inputs-manager/auth-input-state.ts`